tools/releases/dockerfiles/Dockerfile.kumactl

```diff
@@ -1,22 +1,9 @@
-ARG BASE_IMAGE_ARCH=amd64
-FROM --platform=linux/$BASE_IMAGE_ARCH alpine:3.16.3
-
+FROM kumahq/base-debian11:1.0.0
 ARG ARCH
 
-RUN apk add --no-cache curl
-
-ADD /build/artifacts-linux-$ARCH/kumactl/kumactl /usr/bin
-
-COPY /tools/releases/templates/LICENSE \
-    /tools/releases/templates/README \
-    /kuma/
-
 COPY /tools/releases/templates/NOTICE-kumactl /kuma/NOTICE
+COPY /build/artifacts-linux-$ARCH/kumactl/kumactl /usr/bin/
+# copy static curl
+COPY --from=tarampampam/curl:7.78.0 /bin/curl /usr/bin/curl
```

Comment:
this image comes from nowhere trusted. Maybe we can find better?

Reply:
Official curl docs don't build static version of curl, so we have following options:
- use `alpine` (with or without 3rd base image)
- build static curl ourselves (like [this](https://github.com/tarampampam/curl-docker/blob/master/Dockerfile))
- don't include curl?

Reply:
Can't you just copy curl from alpine? Instead of a random image we don't know the origin of?

Reply:
curl from alpine (or official `curlimages/curl`) requires: 
```
        /lib/ld-musl-aarch64.so.1 (0xffffbd72d000)
        libcurl.so.4 => /usr/lib/libcurl.so.4 (0xffffbd638000)
        libz.so.1 => /lib/libz.so.1 (0xffffbd611000)
        libc.musl-aarch64.so.1 => /lib/ld-musl-aarch64.so.1 (0xffffbd72d000)
        libnghttp2.so.14 => /usr/lib/libnghttp2.so.14 (0xffffbd5dc000)
        libssh2.so.1 => /usr/lib/libssh2.so.1 (0xffffbd594000)
        libssl.so.1.1 => /lib/libssl.so.1.1 (0xffffbd505000)
        libcrypto.so.1.1 => /lib/libcrypto.so.1.1 (0xffffbd2b5000)
        libbrotlidec.so.1 => /usr/lib/libbrotlidec.so.1 (0xffffbd29b000)
        libbrotlicommon.so.1 => /usr/lib/libbrotlicommon.so.1 (0xffffbd26a000)
```

so we either copy all of them or we rely on a statically built binary or have alpine as base that we can `apk add --no-cache curl`.

Reply:
Any idea why curl is there btw?

Reply:
I don't think it's needed anywhere by our code, I think it was just added so people can more easily debug things. Which brings me to one more question - @lobkovilya pointed out that we use "debug" versions of distroless images just to have some debugging tools there. My question is, why can't we just use `kubectl debug` for `k8s` and `docker run --pid container:<target> --network container:<target>` or `docker exec` with mount? That way we could just get rid of both things.

Reply:
So the enterprise version does not even have `curl` installed so I think we can get rid of this as well.